### PR TITLE
[test]upgrade api test

### PR DESCRIPTION
**Pull request by FloatXD**
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: test/e2e/API_Performance_Test/api_performance_tests.go [hunk 1]
```diff
@@ -377,16 +377,24 @@ var _ = ginkgo.Describe("localstorage volume test ", ginkgo.Label("api"), func()
...
-			result2, err := utils.RunInLinux(" grep -c '\"Failure Count\": \"0\"'  hwamei_stats.json")
+			result1, err := utils.RunInLinux(" grep -c '\"Failure Count\"' hwamei_test_report* ")
+			result2, err := utils.RunInLinux(" grep -c '\"Failure Count\": \"0\"'  hwamei_test_report*")
 			if err != nil {
 				logrus.Error(err)
 			}
 			gomega.Expect(result1).To(gomega.Equal(result2))
 
 		})
+		ginkgo.It("Upload Report", func() {
```
Comment (SSmallMonster) [inline test/e2e/API_Performance_Test/api_performance_tests.go:390]: is this report can be accessed from public? if can, comment the report address here may be better

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -46,5 +46,5 @@ jobs:
           fetch-depth: 0
       - name: pr test
         env:
-          E2E_TESTING_LEVEL: "pr-e2e"
-        run: make pr-test
+          E2E_TESTING_LEVEL: "api"
+        run: make e2e-test
```
Comment (SSmallMonster) [inline .github/workflows/pr.yml:50]: is this change right? why action `pr-e2e` is changed to `api`

Comment (FloatXD) [reply .github/workflows/pr.yml]: This is just for debugging

File: test/e2e/API_Performance_Test/api_performance_tests.go (hunk 1 above)
Comment (FloatXD) [reply test/e2e/API_Performance_Test/api_performance_tests.go:392]: We will consider updating this to the public website later.

**Pull request merged**